assets/queries/cloudFormation/geo_restriction_disable/test/negative.yaml

```diff
@@ -0,0 +1,15 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Resources:
+  myDistribution:
+    Type: AWS::CloudFront::Distribution
+    Properties:
+      DistributionConfig:
+        Logging:
+          IncludeCookies: 'false'
+          Bucket: mylogs.s3.amazonaws.com
+          Prefix: myprefix
+        Restrictions:
+          GeoRestriction:
+            RestrictionType: whitelist
```

Comment:
We need to add locations to be a valid sample:
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-cloudfront-distribution-georestriction.html
This is needed tell what locations are a whitelist (include) or blacklist (exclude)

Reply:
after a second reading you are correct. this field will only be applied to a location. Otherwise it will not be configured with GeoRestriction!